simmechanics/data/icub3/conf/gazebo_icub_torso.ini

```diff
@@ -37,9 +37,9 @@ max_damping   100.0  100.0  100.0
 [POSITION_CONTROL]
 controlUnits  metric_units
 controlLaw    joint_pid_gazebo_v1
-kp            40.0  40.0  40.0
-kd            0.35  0.35  0.35
-ki            0.35  0.35  0.35
+kp            70.0  0.17   0.15
+kd            70.0  0.17   0.15
+ki            70.0  0.17   0.15
```

Comment:
Just to double check: is this intentional? For all other limbs kd and ki are smaller then kp, here instead they are equal.

Reply:
Thanks a lot for catching that. Its indeed a typo due to using multiple folders to do the changes and tests. Fixed the changes, and did a quick test with the generated model